### PR TITLE
Add more content to on-site page and feedback route.

### PR DESCRIPTION
**Pull request by anselmh**

### CODE REVIEW

File: lib/views/pages/on-site.html [hunk 1]
```diff
@@ -1,3 +1,40 @@
...
+
+<div class="theme">
+    <blockquote>
+        tl;dr: Don't be a jerk.
+    </blockquote>
+</div>
+
+<h3>Code of Conduct</h3>
+<p>
+    As an inclusive event,
```
Comment (drublic) [inline lib/views/pages/on-site.html]: Maybe add a `<br>` after each line.

Comment (anselmh) [inline lib/views/pages/on-site.html]: Ah, no, this is intended for the diffing here, not for the end-result…

Comment (anselmh) [inline lib/views/pages/on-site.html]: Sorry, to give you a bit more context (was a bit late yesterday already) on what I mean here: I was following the concept of [semantic linebreaks](http://blog.doismellburning.co.uk/semantic-linebreaks/) here, but realize that I did only for this section though…

File: lib/views/pages/on-site.html [hunk 2]
```diff
@@ -1,3 +1,40 @@
...
+    <a href="https://twitter.com/helloanselm">@helloanselm</a> / <a href="https://twitter.com/nightlybuildio">@nightlybuildio</a>
+</p>
+
+<div class="theme">
+    <blockquote>
+        How did you like the Event?
+    </blockquote>
+</div>
+<p>We’d love to know how you experienced this year’s event.</p>
+<p><a class="button" href="/feedback">Submit Feedback</a></p>
```
Comment (ixisio) [inline lib/views/pages/on-site.html]: please `target='blank'` the external link

Comment (anselmh) [inline lib/views/pages/on-site.html]: Why? But if so, we should also add `rel=noopener` to it.

File: lib/views/pages/on-site.html [hunk 3]
```diff
@@ -1,3 +1,40 @@
 <% extend 'lib/views/layouts/default' %>
 
 <h2>On Site</h2>
```
Comment (ixisio) [inline lib/views/pages/on-site.html]: "At the Event" is a bit more meaningful than "on site". Can you please change the Route, template name and Headline.

**Comment by ixisio** [comment]
Can you also create a link in main navigation "At the Event" please. Maybe instead "Buy Tickets" cause we are sold out!?

File: lib/views/pages/on-site.html (hunk 2 above)
Comment (ixisio) [inline lib/views/pages/on-site.html]: Ups, yeah you are right. Sorry!

**Comment by ixisio** [comment]
Well done Mr. @anselmh ✅ 

LGTM

**Comment by anselmh** [comment]
Added more information now.

**Comment by ixisio** [comment]
@anselmh Updated the Section "Breaks, Food and Drinks" with infos of this year:

> We will have several breaks between our talks. We will serve drinks before, during and after the event and due to a special deal with one of our sponsors (Nexum), we are able to serve you some fresh brewed local craft beer by Bierlager for free during the event.
> 
> Thanks to our sponsor Michael Page, we also have some food for you. It should be enough but please don’t take two or more servings at the beginning so everyone gets one.

**Comment by ixisio** [comment]
@anselmh Also added hashes to each at-event section to send direct links

**Comment by ixisio** [comment]
@anselmh Ready to merge 👍

**Pull request merged**